libs/libapi/src/libapi/config.py

```diff
@@ -39,6 +39,34 @@ def from_env(cls) -> "UvicornConfig":
 API_MAX_AGE_SHORT = 10  # 10 seconds
 
 
+CACHED_ASSETS_S3_BUCKET = "hf-datasets-server-statics"
```

Comment:
would it make sense to have one S3 bucket for assets, and another one for cached-assets? Maybe it's necessary to apply a different TTL policy for them.

If so, maybe change the bucket name to `hf-datasets-server-cached-assets` and change or remove the folder name , below

Reply:
According to what we discussed with @rtrompier, one bucket is enough since applying TTL policy at folder level will be possible.
wdyt?

Reply:
OK! And is it easy to get the size (number of files, total number of bytes) per directory too?

Reply:
I think so, I was able to see details by folder 
![Screenshot from 2023-09-08 08-26-33](https://github.com/huggingface/datasets-server/assets/5564745/0d871de4-7921-440a-8a0b-203b9577a3be)